src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<Fruit, Integer> fruitInventory = new HashMap<>();
```

Comment:
The `fruitInventory` map should be private to follow the encapsulation principle. Public static fields can be modified from anywhere in the codebase, which can lead to bugs that are hard to track down. Consider making it private and providing access through methods.